pages.ja/common/bc.md

```diff
@@ -0,0 +1,29 @@
+# bc
+
+> 任意の精度で計算を行える言語です。
+> `dc`も参照してください。
+> 詳細はこちら: <https://manned.org/man/bc.1>.
+
+- 対話モードのセッションを開始する:
+
+`bc`
+
+- 標準数学ライブラリを有効化して対話モードのセッションを開始する:
+
+`bc --mathlib`
+
+- 式を計算する:
+
+`echo '{{5 / 3}}' | bc`
+
+- スクリプトを実行する:
+
+`bc {{path/to/script.bc}}`
```

Comment:
Thanks for your contribution, can you translate this path placeholder too?

Reply:
Thanks for your comment. I tried to translate it.